src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ReactNode, useState } from 'react';
+import TestBlock from '../TestBlock';
+import {
+  AccordionHeader,
+  AccordionHeaderContent,
+  AccordionItem,
+  AccordionRoot,
+  AddIcon,
+  CancelIcon,
+  CheckCircleIcon,
+  RemoveIcon,
+  StyledAccordionContent,
+  StyledAccordionTrigger
+} from './style';
+
+interface Props {
+  children: ReactNode;
+  'data-state'?: string;
+  failed?: string[];
+  value?: string;
+}
+
+const TestFileView: React.FC = () => {
+  const [failed, setFailed] = useState(['item-1', 'item-2']);
+
+  return (
+    <AccordionRoot type="multiple" value={failed} onValueChange={(data) => setFailed(data)}>
+      <AccordionItem value="item-1">
```

Comment:
Can we create an array of objects and loop over here?

Reply:
As of now the data is static, as I am currently laying out all the UI components